src/page.h

```diff
@@ -2453,7 +2457,7 @@ class page
 
                     mstch::map tx_cd_data {
                             {"no_of_sources"      , static_cast<uint64_t>(no_of_sources)},
-                            {"use_rct"            , tx_cd.use_rct},
+                            {"use_rct"            , false},
```

Comment:
You likely meant true here.

Reply:
fixed -- `use_rct` is no longer used in the templates but I will leave it in with value `true` for now.